roles/uc-ui/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Assert Debian Stretch
+  assert:
+    that: ansible_distribution == 'Debian' and ansible_distribution_major_version == '9'
+
+- name: Install Wazo repo package source
+  include_role:
+    name: debian-repo-wazo
+
+- name: Install xivo-certs
+  apt:
+    name: xivo-certs
+    state: latest
+
+- name: Install nginx
+  apt:
+    name: nginx
+    state: latest
+
+- name: Install xivo-nginx
+  apt:
+    name: xivo-nginx
+    state: latest
+  notify:
+    - restart nginx
+
+- name: Install custom certificate for HTTPS
+  copy:
+    src: "{{ engine_api_https_cert }}"
+    dest: /usr/share/xivo-certs/server.crt
+    owner: root
+    group: www-data
+    mode: 0640
+  when: engine_api_https_cert is defined
+  notify:
+    - restart wazo-ui
```

Comment:
Why notify wazo-ui, when it is not installed?

Reply:
when we execute uc-ui/tasks/main.yml, wazo-ui is always installed